src/posix.mak

```diff
@@ -83,6 +83,12 @@ endif
 else
 # Default Warnings
 WARNINGS := -Wno-deprecated -Wstrict-aliasing
+ifeq ($(HOST_CC), clang++)
```

Comment:
This isn't going to work if a path is given. I think it would be better to identify the compiler via `$HOST_CC --version` or something.